### PR TITLE
docs: Match styling of Spectrum web ring

### PR DESCRIPTION
**Pull request by cuberoot**
## Description

Prep for publication by removing the old Spectrum logo and working to more closely match the Spectrum webring sites.

This PR replaces #305

- Change use of Side Nav element to match how they use it in the other sites (multi-level)
- Remove Spectrum logo and add Adobe logo
- Add link to Spectrum main site

###

Lots more to do, of course, but this might be enough to get the first public release when we reconvene with their team after the holiday.

## Motivation and Context

Spectrum teams asked for these changes

## How Has This Been Tested?

Visual inspection of doc pages

## Screenshots (if appropriate):

Screen Shot 2019-11-25 at 6 32 52 PM

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: documentation/src/components/adobe-logo.ts [hunk 1]
```diff
@@ -25,31 +25,23 @@ class SpectrumLogo extends LitElement {
         return html`
             <svg
                 version="1.1"
-                id="logosvg"
-                width="${this.size}"
-                height="${this.size}"
+                id="Layer_1"
```
Comment (Westbrook) [inline documentation/src/components/adobe-logo.ts]: Can we remove the id?

File: documentation/src/components/adobe-logo.ts [hunk 2]
```diff
@@ -25,31 +25,23 @@ class SpectrumLogo extends LitElement {
...
-                    className="tier2"
-                    fill="#999999"
-                    d="M188.3,43.5L150,67.2l-38.3-23.7L88.3,57.9l61.7,38l61.4-38L188.3,43.5z"
-                ></path>
-                <path
-                    className="tier1"
-                    fill="#C7C7C7"
-                    d="M150,0l-38.5,23.7l38.3,23.7L188,23.7L150,0z"
-                ></path>
+                    class="st0"
```
Comment (Westbrook) [inline documentation/src/components/adobe-logo.ts]: Can we collapse the style into a `fill="#e1251b"` attribute instead?

File: documentation/src/components/home.ts [hunk 3]
```diff
@@ -12,15 +12,18 @@ class HomeElement extends LayoutElement {
         return html`
             <section id="hero">
                 <div class="spectrum-Article">
-                    <docs-spectrum-logo size="128px"></docs-spectrum-logo>
                     <h1 class="spectrum-Heading1--display">
                         Spectrum Web Components
```
Comment (Westbrook) [inline documentation/src/components/home.ts]: I think we should say "Meet Spectrum Web Components", like other members of the ring.

**Comment by cuberoot** [comment]
@Westbrook I have made those fixes. Thanks! I have never had anyone review the internals of a SVG before :-) Now our version is better than Spectrum CSS's!

I have a few more changes to the docs in the pipe, but I will do a followup.

**Pull request merged**